### DISCUSSION TITLE
Starting working time

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/Paid-plugins#151] [Questions & Answers] [answered]
Hello,

i assigned a task to one of my users. Now i started to track time on this task with "Bearbeitung beginnen"
Now the time is tracked on the assignee of the task and not my account.
Is this how it should be?
Can i not track time on tasks assigned to other users?

THX Niels

**Comment by pbksol** [comment, marked as answer]
As said the other day… If you assign a task to a user and then start processing the timer is started on behalf of that user. You need to unassign the task and then reassign to you to make it start a timer for you. Reason: if a user is already assigned to a ticket, there's no reason for somebody else to work on that task. If someone else should work on that task, it needs to get reassigned.

If you don't assign a user to a task, everybody who starts processing the task will have their own timer started.

**Comment by niels-przybilla** [comment]
Hello Pascal,

many thanks for clarification. It was not so intuitive.
And actually usually as i see :-) I am working together with another team member on the same task :-)

Best wishes

Niels

**Comment by pbksol** [comment]
If you assign the task to the team and not one single user, every team member can start their own timers when working on that task.

**Reply by kevinpapst** [reply]
And then I am open to change requests. The way it is built is ONLY my view on how it should work.
This specific behavior for example was never discussed or at least I can't remember it.

**Reply by niels-przybilla** [reply]
Thank you for being open to suggestions.
